src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -185,7 +185,6 @@
 		01BA06C1261F384A00237DD8 /* WarnOthersReminderTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 948AFE5E2552F6F60019579A /* WarnOthersReminderTests.swift */; };
 		01BA06D3261F41E100237DD8 /* ExposureSubmissionServiceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = CDF27BD2246ADBA70044D32B /* ExposureSubmissionServiceTests.swift */; };
 		01BA06F2261F536100237DD8 /* FakeRequestServiceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 01BA06E3261F52F500237DD8 /* FakeRequestServiceTests.swift */; };
-		01BA06F6261F5A0500237DD8 /* CoronaTestServiceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 01BA06C5261F417700237DD8 /* CoronaTestServiceTests.swift */; };
```

Comment:
I think now the wrong file is gone. `CoronaTestServiceTests.swift` is the new one to keep that was also included in the project, `CoronaServiceTests.swift` (missing the "Test" after "Corona") is the one we can remove, because it has the old name and is only in the file system, not included in the project file.